creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -0,0 +1,114 @@
+const redisClient = require('../redis')
+const { logger: genericLogger } = require('../logging')
+
+const DailyRedisKeyExpirationSec = 90 /* days */ * 24 /* hr */ * 60 /* min */ * 60 /* s */
+
+class SecondarySyncHealthTracker {
+  static async recordSuccess (secondary, wallet, syncType, logContext = {}) {
+    await SecondarySyncHealthTracker._recordSyncRequestOutcome(
+      secondary, wallet, syncType, true, logContext
+    )
+  }
+
+  static async recordFailure (secondary, wallet, syncType, logContext = {}) {
+    await SecondarySyncHealthTracker._recordSyncRequestOutcome(
+      secondary, wallet, syncType, false, logContext
+    )
+  }
+
+  static async _recordSyncRequestOutcome (secondary, wallet, syncType, success, logContext) {
```

Comment:
can we add defaults to success or force an existence check?

Reply:
yep